docker/docker-compose.yml-tmpl

```diff
@@ -27,15 +27,14 @@ services:
   hdfs:
     image: tdm/hdfs
     ports:
-      - "8020:8020"
-      - "8042:8042"
-      - "8088:8088"
-      - "9000:9000"
-      - "10020:10020"
-      - "19888:19888"
-      - "9866:9866"
-      - "9867:9867"
-      - "9870:9870"
-      - "9864:9864"
-      - "9868:9868"
-  
+      - "8020"
```

Comment:
This list actually includes some non-HDFS ports (it was probably copied over from a full Hadoop image). Out of the ports that do belong to HDFS, we can omit the ones we're not using (e.g., secondary name node). This leaves 8020, 9000, 9870, 9866, 9867, 9864. 9870 and 9864 belong to the web UI, so we could leave them exposed.

Reply:
done